misc/gce-federation/README.md

```diff
@@ -0,0 +1,51 @@
+=== GCE quickstart
+cat << EOF > inventory/gce.ini
+gce_service_account_email_address = <service account email address>
+gce_service_account_pem_file_path = <path to service account pem file>
+gce_project_id = <gce project id>
+EOF
+
+export FEDERATION_ID=<my federation id>
+export FEDERATION_DNS_ZONE=<my.federation.dns.zone >
+export GCE_CREDS_FILE=<path to gce credentials json file>
```

Comment:
I created the credentials file with the gcloud utility, I'll provide more info in a followup.